jwql/website/apps/jwql/static/js/jwql.js

```diff
@@ -474,6 +473,12 @@ function update_thumbnail_array(data) {
 
         // Build div content
         content = '<div class="thumbnail" detector="' + filename_dict.detector + '" proposal="' + filename_dict.program_id + '" file_root="' + rootname + '", exp_start="' + file.expstart + '">';
+        if (data.inst!="all") {
+            content += '<a href="/' + data.inst + '/' + rootname + '/">';
+        } else {
+            content += '<a href="/' + filename_dict.inst + '/' + rootname + '/">';
+        }
+        console.log(data.inst);
```

Comment:
I realize it is a little messy looking. I had put the instrument within the filename_dict for the thumbnails generated when multiple instruments are present, so I thought this may be a good way to add it to the content. Otherwise, there is only an option for one instrument for the entire array of thumbnails that includes those from multiple instruments. Do you by chance have another idea for how we could access this?

Reply:
Sorry!  I meant to just remove the `console.log` statement.

Reply:
Oh yes, absolutely.